kolibri/plugins/learn/assets/src/views/content-card-carousel/index.vue

```diff
@@ -0,0 +1,222 @@
+<template>
+
+  <section class="content-carousel">
+
+    <div class="content-carousel-details">
+      <header v-if="header" class="content-carousel-details-header">
+        <h2> {{header}} </h2>
+        <span v-if="subheader"> {{subheader}} </span>
+      </header>
+
+      <icon-button
+        :text="$tr('viewAllButtonLabel')"
+        v-if="showViewAll"
+        @click="$emit('viewAllClicked')"
+        class="content-carousel-details-view-all" />
+    </div>
+
+
+    <div class="content-carousel-set">
+
+      <transition :name="animation">
+        <div :key="currentSetIndex" :style="widthOfCarousel" class="content-carousel-cards">
+          <slot
```

Comment:
@indirectlylit @christianmemije @rtibbles Saw the opportunity for scoped slots to minimize refactoring of the filter code. 

Pass in the info you want the parent to have access to within the slot like a prop here